bootstrap-app/src/cmr/bootstrap/services/bootstrap_service.clj

```diff
@@ -15,26 +12,29 @@
     [cmr.indexer.system :as indexer-system]
     [cmr.transmit.index-set :as index-set]))
 
+(def request-type->dispatcher
+  "A map of request types to which dispatcher to use for asynchronous requests."
+  {:migrate-provider :core-async-dispatcher
+   :migrate-collection :core-async-dispatcher
+   :index-provider :core-async-dispatcher
```

Comment:
For this ticket (CMR-4343) I plan to just change the index-provider type to use the message queue (in my next PR). I will write additional tickets for changing over anything else that could benefit from horizontal scaling using a message queue.